index.html

```diff
@@ -2316,11 +2316,39 @@ <h2>Privacy Considerations</h2>
       <section>
         <h3>Unlinkability</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are digital signature
-schemes that provide uncorrelatable digital signatures.
+        <p>
+When the contents of a digitally signed payload contains a correlatable
+identifier, that identifier can be used for tracking purposes by a verifier,
+which might lead to a violation of an expected level of privacy. An example of
+such an identifier is a shopping loyalty card number. However, some of these
+identifiers might not be immediately apparent, such as a static digital
+signature or a cryptographic hash of an image, which can also be used as a
+correlatable identifier.
+        </p>
+
+        <p>
+It is possible to create a digitally signed payload that does not have any
+correlatable tracking identifiers while also providing some level of assurance
+that the payload is trustworthy for a given interaction. This characteristic is
+called <dfn>unlinkability</dfn> which ensures that no correlatable identifiers
+are used in a digitally signed payload while providing some level of trust.
+        </p>
+
+        <p>
+There are at least two aspects to providing unlinkability. The first aspect is
+to 1) ensure that every data field used in the message is never repeated in a
+future message, or 2) ensure that any repeated field(s) provide adequate herd
+privacy such that it becomes practically impossible to correlate the entity that
+expects some level of privacy in the interaction.
+        </p>
+
+        <p>
+A variety of methods can be used to achieve unlinkability. These methods
+include the utilization of single use bearer tokens, carefully crafted
+messages that utilize attributes that ensure an adequate level of herd privacy,
+and the use of cryptosuites that enable the entity presenting a message to
+re-generate new signatures while not compromising the trust in the message
```

Comment:
```suggestion
regenerate new signatures while not compromising the trust in the message
```

Reply:
Upon re-reading, neither `regenerate new signatures` nor `re-generate new signatures` works. It should be either `generate new signatures` or `regenerate signatures`, of which I'm inclined toward the former.